PLAN.md

```
@@ -0,0 +1,438 @@
+          return input.items.length;
+        });
+    });
+  });
+});
+```
+
+## Implementation Timeline
+
+### Day-by-Day Breakdown
+
+**Day 1: API Design & Type Tests**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)